tests/lit-tests/math-lib-value.ispc

```diff
@@ -0,0 +1,40 @@
+// RUN: %{ispc} %s --target=host --nowrap 2>&1 --debug-phase=2:2 -o t.o | FileCheck %s --check-prefix=CHECK-ISPC
```

Comment:
Please add some description what this test is doing and where are defined libs values.

Reply:
Done